Tests/common_BioSQL.py

```diff
@@ -972,9 +971,8 @@ def test_server(self):
     def test_del_db_items(self):
         """Check all associated data is deleted from an item."""
         db = self.db
-        items = list(db.values())
         keys = list(db)
-        length = len(items)
+        self.assertEqual(len(keys), len(list(db.values())))
```

Comment:
Can this test fail? I would expect that the number of keys is always equal to the number of values.

Reply:
I didn't actually double check the implementation, the style error was assigning things to otherwise unused variables. I assumed the original intent was to invoke the iteration and values methods, thus this alternative (affects a couple of lines - not just the one highlighted here in GitHub).

But yes, the current implementation of ``.values()`` is a loop over the ``__iter__`` method so this assert ought never to fail.